daemon/start.go

```diff
@@ -11,6 +11,15 @@ func (daemon *Daemon) ContainerStart(job *engine.Job) error {
 	if len(job.Args) < 1 {
 		return fmt.Errorf("Usage: %s container_id", job.Name)
 	}
+
+	config := runconfig.ContainerConfigFromJob(job)
+	hostConfig := runconfig.ContainerHostConfigFromJob(job)
```

Comment:
Emm ... I think my peer misunderstanded your idea here, we'll re-commit and squash it